playground/mocks/data/idp/idx/authenticator-verification-okta-verify-send-push-autoChallenge.json

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
and following my other comments, this should be "authenticator-verification-data-okta-verify-push.json" (similar to the other file with `authenticator-verification-data` remediation form, but this one does not have `autoChallenge` on

Reply:
have renamed this to `authenticator-verification-data-okta-verify-push-autoChallenge-off` and 
renamed `authenticator-verification-okta-verify-push-autoChallenge` to `authenticator-verification-okta-verify-push-autoChallenge-on`